python/aoa.py

```diff
@@ -0,0 +1,48 @@
+import jsbsim
```

Comment:
The jsbsim/python directory is for files related to building the python wrapper module for JSBSim, not for semi-random useful python scripts. It would potentially be useful to include some of these sorts of scripts as an example of what the python wrapper can be used for, but we should create a separate directory for them.

Reply:
where can I include them?